integration-tests/contracts/contract_loader.go

```diff
@@ -2,6 +2,7 @@ package contracts
 
 import (
 	"errors"
+
```

Comment:
This, and below files seem like they don't need to change for your PR.
Are you using some default tool or IDE that is automatically making these below changes?